rvic/core/read_forcing.py

```diff
@@ -19,6 +21,19 @@
 log = getLogger(LOG_NAME)
 # -------------------------------------------------------------------- #
 
+# -------------------------------------------------------------------- #
+# create Listener to send timestamp information to connecting Client
+def send_timestamp(timestamp, event):
+    address = ("0.0.0.0", 5005)
+    with Listener(address) as listener:
+        listener._listener._socket.settimeout(0.5) # Ensure that listener is not always waiting for connection in case there is no Client
+        while not event.is_set():
+            try:
+                with listener.accept() as conn:
```

Comment:
I believe that this uses blocking I/O which is non-ideal. Most of the time, it's going to be sitting there doing nothing, blocking execution of anything else. Now, maybe that doesn't matter if it's running in a thread? I'm not sure. But if it's possible to write this using cooperative multitasking (like with Python's `await` keyword), that might be better. See: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.run_in_executor